### ISSUE TITLE
Mosquitto broker "The ordinal 340" error

### ISSUE THREAD

**Issue by spyhack225** [eclipse/mosquitto#276]
Hi everyone i try to install mosquitto on windows 8 but i have error, i already add the 2 dll in the installation directory.
![ice_screenshot_20160926-070233](https://cloud.githubusercontent.com/assets/19724890/18824077/629731d4-83b7-11e6-83ea-01d75af4d12d.png)

**Comment by ralight** [comment]
The most likely cause for this is that you haven't got the Visual Studio 2013 x86 Redistributable:

https://www.microsoft.com/en-GB/download/details.aspx?id=40784

The other possibility is that one of the other dlls needs updating.

**Comment by ralight** [comment]
Another possibility is that you need version 1.0.2* of openssl, not 1.1.0.

I'm closing this because there haven't been any updates for a while.

**Comment by i3laze** [comment]
That’s true, OpenSSL v1.0.2 x86 .DLLs (Light binary will fit) are the exact ones needed to run:
http://slproweb.com/download/Win32OpenSSL_Light-1_0_2o.exe
